platform/backend/repository/api/src/main/kotlin/io/hamal/repository/api/Func.kt

```diff
@@ -48,18 +46,21 @@ interface FuncCmdRepository : CmdRepository {
         val namespaceId: NamespaceId,
         val name: FuncName,
         val inputs: FuncInputs,
-        val code: FuncCode  // FIXME-53 this is bad now - because it contains deployed version
-        // FIXME-53 val codeId: CodeId
-        // FIXME-53 val codeVersion: CodeVersion
+        val codeId: CodeId,
+        val codeVersion: CodeVersion
     )
 
     data class UpdateCmd(
         val id: CmdId,
         val name: FuncName? = null,
         val inputs: FuncInputs? = null,
-        val code: FuncCode? = null // FIXME-53 this is bad now - because it contains deployed version same
-        // FIXME-53 val codeId: CodeId
-        // FIXME-53 val codeVersion: CodeVersion
+        val codeId: CodeId? = null,
```

Comment:
Looking at this now - I think we can get rid of the codeId for the update command, because it will never change only the code version keeps changing